src/topology/algebra/module/basic.lean

```diff
@@ -2038,6 +2038,18 @@ end
 
 end
 
+variables [ring R] [add_comm_group M] [topological_add_group M] [module R M]
+
+/-- Given an invertible operator, multiplying it by its inverse gives the identity. -/
+lemma inv_mul_self  (T : M →L[R] M) [invertible T] : T.inverse * T = 1 :=
```

Comment:
Usually we only use `invertible` if you use `\frac1` (aka `inv_of`) in the statement. Since you don't, can you use `is_unit T` instead?

Reply:
Wouldn't it be easier to use `\frac1` instead of `inv_of` everywhere?

Reply:
`inv_of` is to `\frac1` as `mul` is to `*`. We use one in lemma names, but the other is the operator.